system-variables.md

```diff
@@ -1094,6 +1094,14 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 ### tidb_ddl_enable_fast_reorg <span class="version-mark">New in v6.3.0</span>
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> To improve the speed for index creation using this variable, make sure that your TiDB cluster is hosted on AWS and your TiKV node size is at least 8 vCPU and 16 GiB. For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), this feature is unavailable.
```

Comment:
"your TiKV node size is at least 8 vCPU and 16 GiB" is just a placeholder to be updated

Reply:
@benmaoer Would you please help confirm? Thanks.